libs/fslock/lock_unix.go

```diff
@@ -0,0 +1,48 @@
+//go:build darwin || freebsd || linux
```

Comment:
Are we keeping track of which systems we will support?

Reply:
Good question, I think we need to have some sort of ongoing `README` updating -- this would be something good to add to the README. I will make an issue for the README, will be more relevant when we have the binary and need install instructions, etc.

Reply:
#85